core/base/src/TApplication.cxx

```diff
@@ -1073,6 +1085,68 @@ void TApplication::Help(const char *line)
+         Printf("   CTRL+Arrow Right    : jump to next word" FOOTNOTE);
+         Printf("   CTRL+B              : move cursor left");
+         Printf("   CTRL+C              : send kSigInt interrupt signal");
+         Printf("   CTRL+D              : remove next character (or exit ROOT if empty line)");
+         Printf("   CTRL+Del            : remove next word" FOOTNOTE);
+         Printf("   CTRL+E              : move cursor to end of line");
+         Printf("   CTRL+F              : move cursor right");
+         Printf("   CTRL+G              : move cursor right");
+         Printf("   CTRL+H              : remove previous character");
+         Printf("   CTRL+I              : autocomplete command or print suggestions");
+         Printf("   CTRL+J              : execute command");
+         Printf("   CTRL+K              : remove all characters between cursor and end of line");
+         Printf("   CTRL+L              : clear screen");
+         Printf("   CTRL+M              : execute command");
+         Printf("   CTRL+N              : navigate downwards in command history");
+         Printf("   CTRL+O              : select command of the string-based history search (also Enter)");
+         Printf("   CTRL+P              : navigate upwards in command history");
+         Printf("   CTRL+Q              : ");
+         Printf("   CTRL+R              : reverse string-based history search");
+         Printf("   CTRL+T              : clear screen");
+         Printf("   CTRL+U              : remove all characters between cursor and start of line");
+         Printf("   CTRL+W              : delete previous word");
+         Printf("   CTRL+Y              : paste clipboard content (also CTRL+Shift+V)");
+         Printf("   CTRL+Z              : send kSigStop pause job signal");
+         Printf("   Ctrl+_              : undo command");
+         Printf("   Del                 : remove previous character"); // Del is 0x7F on macOS
+         Printf("   End                 : move cursor to end of line");
+         Printf("   Enter               : execute command");
+         Printf("   Esc, B              : jump to previous word");
+         Printf("   Esc, Backspace      : remove previous word");
+         Printf("   Esc, C              : character to upper and jump to next word");
+         Printf("   Esc, CTRL+H         : remove previous word");
+         Printf("   Esc, D              : remove next word");
+         Printf("   Esc, Del            : remove previous word"); // Del is 0x7F on macOS
+         Printf("   Esc, CTRL+Del       : remove previous word"); // Del is 0x7F on macOS
+         Printf("   Esc, F              : jump to next word");
+         Printf("   Esc, L              : word to lower and jump to it")
+         Printf("   Esc, U              : word to upper and jump to it")
+         Printf("   Home                : move cursor to beginning of line");
```

Comment:
Home ? which key is that ? I do not have it on Mac Keyboard

Reply:
Fn+Left. Changed now.